### PR TITLE
[GH-18195] Remove all instances of zz + model.NewId() in storetest/us…

### PR DESCRIPTION
**Pull request by gozeloglu**
#### Summary

Remove all instances of "zz" + model.NewId() in storetest/user_store.go
#### Ticket Link

Fixes https://github.com/mattermost/mattermost-server/issues/18195
#### Release Note

```release-note

```

### CODE REVIEW

File: store/storetest/user_store.go [hunk 1]
```diff
@@ -2764,7 +2764,7 @@ func testUserStoreSearchNotInChannel(t *testing.T, ss store.Store) {
 	ch1 := model.Channel{
 		TeamId:      tid,
 		DisplayName: "NameName",
-		Name:        "zz" + model.NewId() + "b",
+		Name:        NewTestId() + "b",
```
Comment (agnivade) [inline store/storetest/user_store.go:2767]: Please remove the `b` here and everywhere else as well.

Comment (gozeloglu) [reply store/storetest/user_store.go]: I did.

**Review by agnivade — APPROVED** [review]
Thank you @gozeloglu !

**Comment by gozeloglu** [comment]
Hi @isacikgoz. Can you review my changes? Thanks in advance.

**Review by isacikgoz — APPROVED** [review]
Thanks @gozeloglu 👍

**Pull request merged**